msteams-platform/m365-apps/using-teams-client-sdk-preview.md

```diff
@@ -0,0 +1,365 @@
+
+1. A command to update your manifest schema
+1. A command to update your SDK references and call sites
+
+These are the two key updates you'll need to run a Teams personal tab app in other Microsoft 365 applications.
+
+### 2. Updating the manifest
+
+# [Teams Toolkit](#tab/manifest-teams-toolkit)
+
+1. Open the *Command palette*: `Ctrl+Shift+P`
+1. Run the `Teams: Upgrade Teams manifest to support Outlook and Office apps` command and select your app manifest file. Changes will be made in place.
+
+# [Manual steps](#tab/manifest-manual)
+
+Open your Teams app manifest and update the `$schema` and `manifestVersion` with the following values:
+
+```json
+{
+    "$schema" : "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
+    "manifestVersion" : "m365DevPreview"
+}
+```
+---
+
+If you used Teams Toolkit to create your personal app, you can also use it to validate the changes to your manifest file and identify any errors. Open the command palette `Ctrl+Shift+P` and find **Teams: Validate manifest file** or select the option from the Deployment menu of the Teams Toolkit (look for the Teams icon on the left side of Visual Studio Code).
+
+:::image type="content" source="images/toolkit-validate-manifest-file.png" alt-text="Teams Toolkit 'Validate manifest file' option under 'Deployment' menu":::
+
+### 2. Update SDK references
+
+To run in Outlook and Office, your app will need to depend on the [npm package](https://www.npmjs.com/package/@microsoft/teams-js/v/2.0.0-beta.1) `@microsoft/teams-js@2.0.0-beta.1` or higher. To perform these steps manually, and for more information on the API changes, see the following sections on [Callbacks converted to promises](#callbacks-converted-to-promises) and [APIs organized into capabilities](#apis-organized-into-capabilities).
+
+1. Ensure you have [Teams Toolkit](https://aka.ms/teams-toolkit) `v2.10.0` or later
+1. Open the *Command palette*: `Ctrl+Shift+P`
+1. Run the command `Teams: Upgrade Teams JS SDK references to support Outlook and Office apps`
+
+Upon completion, the utility will have updated your `package.json` file with the TeamsJS SDK v2 Preview (`@microsoft/teams-js@2.0.0-beta.1`) dependency, and your `*.js/.ts` and `*.jsx/.tsx` files will be updated with:
+
+> [!div class="checklist"]
+> * `package.json` references to TeamsJS SDK v2 Preview
+> * Import statements for TeamsJS SDK v2 Preview
+> * [Function, Enum, and Interface calls](#apis-organized-into-capabilities) to TeamsJS SDK v2 Preview
+> * `TODO` comment reminders to review areas that might be impacted by [Context](#updates-to-the-context-interface) interface changes
+> * `TODO` comment reminders to ensure [conversion to promises functions from callback style functions](#callbacks-converted-to-promises) has gone well at every call site the tool found
+
+> [!IMPORTANT]
+> Be sure to review all `TODO` items flagged by the tool.
```

Comment:
"Ensure to review . . . " doesn't sound correct to my ear. I actually don't think we need this important callout, because its implied by the checklist above (and too many IMPORTANT notes dilute importance). Removed it (and also the same in personal-tab.md)